### PR TITLE
chore: remove releases and upgrades from CODEOWNERS

### PR DESCRIPTION
**Pull request by erickzhao**
makes it so that PRs to this repo are not blocked by releases and upgrades approval.

### CODE REVIEW

**Review by MarshallOfSound — APPROVED** [review]

**Review by dsanders11 — CHANGES_REQUESTED** [review]
Doesn't this just mean that those WGs aren't able to approve?

**Comment by MarshallOfSound** [comment]
Rock and hard place, we shouldn't require 3 working groups approval to make a change to this module.  It should be any of the three, we've done this in the past with `@electron/patch-owners` by making a new formation group in `.permissions` and assigning them as the code owners but that's a lot of effort for every possible permutation of WG code owners

**Comment by dsanders11** [comment]
@MarshallOfSound, it *is* any of the three currently. GitHub only enforces a single CODEOWNER need approve if they're all on the same line, which is true here.

**Pull request closed without merging**
